scripts/data_access/create_data_access_view.sql

```diff
@@ -0,0 +1,132 @@
+#standardSQL
+
+--
+-- This is setup for storage.googleapis.com and bigquery.googleapis.com
+--
+-- This is for examining access to bigquery datasets and buckets.
+-- 
+-- Table and object-level is available in the logs (and more!), but
+-- this creates confusion for a dashboard.
+-- 
+CREATE OR REPLACE VIEW data_access.audit_summary AS
+WITH
+  -- Pull out Data Access logs
+  DataAccess AS (
+    SELECT
+      -- Hour truncated
+      TIMESTAMP_TRUNC(d.timestamp, HOUR) AS hour,
+      -- Project ID that the access method was called on
+      d.resource.labels.project_id,
+      -- Actor
+      d.protopayload_auditlog.authenticationInfo.principalEmail AS actor,
+      -- Permission used to access data
+      SPLIT(i.permission,'.')[SAFE_OFFSET(0)] AS service,
+      -- Permission used
+      i.permission AS action,
+      -- Whether granted or denied
+      IFNULL(i.granted, FALSE) AS granted,
+      -- Parts of the resource accessed
+      SPLIT(i.resource, '/') AS parts
+    FROM
+      `${PROJECT_ID}.data_access.cloudaudit_googleapis_com_data_access_*` d
+      CROSS JOIN d.protopayload_auditlog.authorizationInfo i
+    WHERE
+      i.resource IS NOT NULL AND
+      d.protopayload_auditlog.serviceName IN ('storage.googleapis.com',
+                                              'bigquery.googleapis.com')
+  )
+SELECT
+  hour,
+  service,
+  actor,
+  -- Translate the action into an operation (READ/WRITE/ADMIN)
+  CASE
+    WHEN service = 'storage' THEN
```

Comment:
Do you think by checking the action value, you can determine the action type? So, maybe simplify the code a little with one level of CASE statement?

Same for service='bigquery' code below.